src/analysis.zig

```diff
@@ -171,6 +182,10 @@ pub fn getChild(tree: *ast.Tree, node: *ast.Node, name: []const u8) ?*ast.Node {
                 const field = child.cast(ast.Node.ContainerField).?;
                 if (std.mem.eql(u8, tree.tokenSlice(field.name_token), name)) return child;
             },
+            .ErrorTag => {
+                const tag = child.cast(ast.Node.ErrorTag).?;
+                if(std.mem.eql(u8, tree.tokenSlice(tag.name_token), name)) return child;
+            },
```

Comment:
What does `getChild` do? It doesn't seem to matter whether `.ContainerField` and `.ErrorTag` are here.

Reply:
`getChild` gets the child of a node by name. This is useful for completion of values that are children of a variable or field's types, of a function's return value. I don't think `.ErrorTag` in `getChild` is necessary, as far as I know.